modules/oe_whitelabel_contact_forms/tests/src/Kernel/ContactFormRenderTest.php

```diff
@@ -112,20 +113,63 @@ public function testContactForm(): void {
     // Assert message success and values.
     $actual = $crawler->filter('div.alert-success');
     $this->assertCount(1, $actual);
-    $dt = $crawler->filter('dt');
-    $dd = $crawler->filter('dd');
-    $this->assertEquals("The sender's name", $dt->eq(0)->text());
-    $this->assertEquals("The sender's email", $dt->eq(1)->text());
-    $this->assertEquals("Subject", $dt->eq(2)->text());
-    $this->assertEquals("Message", $dt->eq(3)->text());
-    $this->assertEquals("Topic", $dt->eq(4)->text());
-    $this->assertEquals("Phone", $dt->eq(5)->text());
-    $this->assertEquals("sender_name", $dd->eq(0)->text());
-    $this->assertEquals("test@example.com", $dd->eq(1)->text());
-    $this->assertEquals("subject", $dd->eq(2)->text());
-    $this->assertEquals("welcome_message", $dd->eq(3)->text());
-    $this->assertEquals("Topic name", $dd->eq(4)->text());
-    $this->assertEquals("0123456", $dd->eq(5)->text());
+
+    $description_lists = $crawler->filter('dl');
+    $description_list_assert = new DescriptionListAssert();
```

Comment:
I want to move all pattern assertion classes that I find into oebt, should I do it in this ticket or prep a new one?

Reply:
They don't need to be moved, they need to be redone XD The ones in whitelabel I think they are mostly wrong.